modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/UpdateArticleMVCActionCommand.java

```diff
@@ -522,8 +522,14 @@ else if (group.hasStagingGroup()) {
 			return Collections.emptyMap();
 		}
 
+		String portletResource = ParamUtil.getString(
+			actionRequest, "portletResource");
+
+		boolean isPortlet = !portletResource.isEmpty();
```

Comment:
using the !Validator.isBlank or Validator.isNotNull(), write :
`

		boolean isPortlet = !Validator.isBlank(ParamUtil.getString(
			actionRequest, "portletResource"));`

Reply:
I've fixed this.